src/assets/drizzle/styles/components/_footer.scss

```diff
@@ -0,0 +1,18 @@
+.#{$app-namespace}-c-footer {
+  background-color: $purple;
+  padding: $space-inset-short-l;
+}
+
+.#{$app-namespace}-c-footer__icons {
+  @media (max-width: $footer-collapse-breakpoint) {
+    margin-top: $space-s;
```

Comment:
We try to provide margin in one vertical direction as best as we can. So, this would preferably be margin on the bottom here or you could take a look at using the Stack object that we have. We generally try to avoid using spark components in the doc site because of their nature of change but Stack being an object is safe.